### ISSUE TITLE
Invalid prop `checked` supplied to `Checkbox`

### ISSUE THREAD

**Issue by VimLeSai** [react-component/checkbox#49]
**Unnecessary Warning**

my code: 

```JS
import CheckBox from 'rc-checkbox'; 
...
 <CheckBox
          color="#29363d"
          checked={historyItem.pdf}
          className="checkbox-design"
        />
```
When i load this compoent is shows `Warning` as below,

`Warning: Failed prop type: Invalid prop 'checked' supplied to 'Checkbox'.
    in Checkbox (created by xyz)`

i hope this will help...

**Comment by wkba** [comment]
@VimLeSai 
Have you solved this question yet?
Please try use defaultChecked, not checked

**Comment by HungNho17** [comment]
I face same issue too

**Comment by afc163** [comment]
It should be fixed in latest version.
